tools/local-network/send-mobilecoind-txs.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
I wonder if this would be better as a Python script. We'd get better structs/encapsulation and cleaner arrays/looping, as well as easier JSON de/serialization

Reply:
Potentially, I think it would add more bloat than is necessary at this point